libraries/abstractions/secure_sockets/test/iot_test_tcp.c

```diff
@@ -423,7 +426,7 @@ typedef struct
 #define tcptestMAX_LOOPS_ECHO_TEST            tcptestMAX_ECHO_TEST_MODES
 
 #define tcptestECHO_TEST_LOW_PRIORITY         tskIDLE_PRIORITY
-#define tcptestECHO_TEST_HIGH_PRIORITY        ( configMAX_PRIORITIES - 1 )
+#define tcptestECHO_TEST_HIGH_PRIORITY        ( configMAX_PRIORITIES - 2 )
```

Comment:
What is the reason for changing this?

Reply:
This comes from here:
https://quip-amazon.com/KNMVAxus311S/Secure-Sockets-Test-Plan-Changes-proposal#cNK9CAbPC6R